tests/ui/iterators/into-iter-on-arrays-2018.stderr

```diff
@@ -55,11 +55,10 @@ help: use `.iter()` instead of `.into_iter()` to avoid ambiguity
    |
 LL |     for _ in [1, 2, 3].iter() {}
    |                        ~~~~
-help: or remove `.into_iter()` to iterate by value
-   |
-LL -     for _ in [1, 2, 3].into_iter() {}
-LL +     for _ in [1, 2, 3] {}
+help: or use `IntoIterator::into_iter(..)` instead of `.into_iter()` to explicitly iterate by value
    |
+LL |     for _ in IntoIterator::into_iter([1, 2, 3]) {}
+   |              ++++++++++++++++++++++++         ~
```

Comment:
Why did this suggestion change?

Reply:
:think: I think I'm doing the for loop desugaring computation wrong

Reply:
I fixed this